hubverse-aws-upload/hubverse-aws-upload.yaml

```diff
@@ -57,15 +57,24 @@ jobs:
           'hub-config'
           'model-abstracts'
           'model-metadata'
+          # temporarily include model-output here
           'model-output'
           'target-data'
         )
         for DIRECTORY in "${hub_directories[@]}"
         do
-          if [ -d "./$DIRECTORY" ]; then aws s3 sync "./$DIRECTORY" "s3://$BUCKET_NAME/$DIRECTORY" --delete; fi
+          if [ -d "./$DIRECTORY" ]
+          then
+            rclone sync \
+              "./$DIRECTORY/" \
+              ":s3,provider=AWS,env_auth:$BUCKET_NAME/$DIRECTORY" \
```

Comment:
This line is an rclone S3 [connection string](https://rclone.org/docs/#connection-strings). `env_auth` instructs rclone to get AWS creds via environment variables. Those AWS env variables are set in the prior `Configure AWS credentials` step.